pkgs/development/python-modules/mkdocs-simple-hooks/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, buildPythonPackage
+, isPy3k
+, fetchFromGitHub
+, mkdocs
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "mkdocs-simple-hooks";
+  version = "0.1.5";
+
+  disabled = !isPy3k;
+
+  src = fetchFromGitHub {
+    owner = "aklajnert";
+    repo = "mkdocs-simple-hooks";
+    rev = "v${version}";
+    hash = "sha256-N6xZjCREjJlhR6f8m65WJswUQv/TTdTbk670+C46UWQ=";
+  };
+
+  propagatedBuildInputs = [
+    mkdocs
+  ];
+
+  nativeCheckInputs = [ pytestCheckHook ];
+
+  pytestFlagsArray = [ "tests.py" ];
+
+  # disable failing tests
+  disabledTests = [
+    "test_no_hooks_defined"
+    "test_no_attribute"
+  ];
+
```

Comment:
```suggestion

  pythonImportsCheck = [ "mkdocs_simple_hooks" ];

```

Reply:
Sorry, I can't add this because it doesn't work. This hook is not meant to be imported. On other distros it doesn't work  either.